packages/springboard/src/types/shared.ts

```diff
@@ -0,0 +1,3 @@
+export type TestingType = 'e2e' | 'component'
```

Comment:
We should look into a `packages/shared` at some point, for things like this. But not right now (imo).